themes/scss/widgets/text-elements.scss

```diff
@@ -43,9 +40,9 @@
 }
 .CDB-Widget-textSmaller { // 10 size
   @include text-overflow();
-  font-size: $sFontSize-smaller;
+  font-size: $sFontSize-small;
   font-weight: $sFontWeight-normal;
-  line-height: $sFontSize-smaller + 3;
+  line-height: $sFontSize-small + 3;
```

Comment:
Name of variable `sFontSize-small` should be written in all lowercase letters with hyphens instead of underscores